### ISSUE TITLE
TTS training from the PPG

### ISSUE THREAD

**Issue by ravi-0841** [guanlongzhao/fac-via-ppg#22]
I am trying to train the Tacotron model with the provided code but it does not seem to be working at all. The model fails to learn anything from the input features and the dataset provided by the authors. Here is how my training and validation loss looks like even after 1000 epochs. 
![train_loss](https://user-images.githubusercontent.com/12909938/205093472-49b53097-abe9-431a-bd43-9a16bd946314.png)
![valid_loss](https://user-images.githubusercontent.com/12909938/205093532-933db228-5cb7-4494-8aca-c6c95b94f46c.png)

I am merely trying to reproduce the work here and have not made any changes to the code.

**Comment by guanlongzhao** [comment]
Can you attach your environment info? E.g., Linux version, PyTorch version, etc. Also which command did you run to train the model? What does the mel-spectrogram on the eval set look like?
